app/modules/entities/components/lib/relative_entities_helpers.js

```diff
@@ -0,0 +1,38 @@
+import { i18n } from '#user/lib/i18n'
+import { inverseLabels } from '#entities/components/lib/claims_helpers'
+
+export function getRelativeEntitiesListLabel ({ property, entity }) {
+  const label = inverseLabels[property]
+  if (label) return i18n(label, { name: entity.label })
+}
+
+export function getRelativeEntitiesProperties (type, mainProperty) {
+  // TODO: add genre relatives
+  let properties = relativeEntitiesPropertiesByType[type]
+  // Omit the property already displayed in the entity browser of a claim layout.
+  if (mainProperty) properties = _.without(properties, mainProperty)
+  return properties || []
+}
+
+export const relativeEntitiesPropertiesByType = {
+  work: [
+    'wdt:P136', // genre
```

Comment:
That would display lists of works that a work as genre? Do we want to encourage this kind of heresy?

Reply:
done in a659873bc3dd7ac860bab51404fa5eaadf5a2db6